src/parser/keyword.c

```diff
@@ -142,6 +142,7 @@ static KEYWORD_RECORD keywords[] = {
   {DAY_MILLISECOND, "DAY_MILLISECOND", 0},
   {DAY_MINUTE, "DAY_MINUTE", 0},
   {DAY_SECOND, "DAY_SECOND", 0},
+  {DBLINK, "DBLINK", 1},
```

Comment:
unreserved ==>  keyword can be used as an identifier, 0 means it is reserved and cannot be used as an identifier, nonzero means it can be.

**Which value would be appropriate to set the dblink keyword to between 0 and 1?**

Reply:
I think it should be set to 0 as reserved key word.